ui/src/pages/AllStories/reducer.ts

```diff
@@ -13,20 +13,23 @@ export interface State {
   selectedRowIds: number[];
   order: "asc" | "desc";
   orderBy: string;
+  tags: string[];
+  filterState: any;
```

Comment:
Let's type this out: 
```suggestion
   filterState: FilterState;
}

interface FilterState {
  visibility: VisibilityType;
  tags: Record<string, boolean>;
};
type VisibilityType = {
  visible: boolean;
  nonVisible: boolean;
};
```